.github/workflows/test_merge_requests.yml

```diff
@@ -150,9 +151,18 @@ jobs:
           platforms: linux/amd64
           cache-from: type=local,src=/tmp/.buildx-cache
 
-      - name: 'Run tests'
-        env:
-          CI: true
-        run: |
+      - name: Run tests
+        run: >
           NAME=mailserver-testing:ci
           make generate-accounts tests
+        env:
+          CI: true
```

Comment:
Why did you revert this?

`run: >` treats it all as a single line, which is fine in this case, but I don't know if there's any benefit vs mult-line `|` usage?

Any reason for moving the `env` back down to the bottom of the step? Is that just a preference? I shifted it above `run` as it seemed more appropriate to have ENV listed before `run`.

I don't mind either of these, I just want to avoid making these changes if we're going to have ping-pong edits whenever we work on workflows :smile:

Reply:
> Why did you revert this?
> 
> `run: >` treats it all as a single line, which is fine in this case, but I don't know if there's any benefit vs mult-line `|` usage?

With `|`, I am not confident whether `make` uses `NAME` when calling the `generate-accounts` and `tests` recipes, as with `>`, I know for a fact that it is the case.

> Any reason for moving the `env` back down to the bottom of the step? Is that just a preference? I shifted it above `run` as it seemed more appropriate to have ENV listed before `run`.
> 
> I don't mind either of these, I just want to avoid making these changes if we're going to have ping-pong edits whenever we work on workflows smile

I looked at https://github.com/docker-mailserver/docker-mailserver/blob/e6f481b59f7f67a7f605f9d97fa6a5ee1de68840/.github/workflows/default_on_push.yml#L56-L61, and wanted to be coherent.

Reply:
In that case, maybe we should just add `NAME` under `env`? Then `run: make generate-accounts tests`? (default shell is bash)

Reply:
Yep, will do :D